domains/stebanportfolio.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "Give a profesional domain to my portfolio",
+    "repo": "https://github.com/fergone03/Portfolio-Desarrollador-Web",
+    "owner": {
+        "username": "fergone03",
+        "email": "[fergonesteban03@gmail.com]"
```

```suggestion
        "email": "fergonesteban03@gmail.com"
```
also suggest you using [OWL](https://owl.is-a.dev) or removing the mail field to avoid receiving random spam or scam mails from users who view the repo